akka-jackson/src/main/resources/reference.conf

```diff
@@ -0,0 +1,55 @@
+############################
+# Akka Jackson Config File #
+############################
+
+# This is the reference config file that contains all the default settings.
+# Make your edits/overrides in your application.conf.
+
+akka.jackson {
```

Comment:
tempted namespace it into Akka.serialization and then the Jackson namespace. at least to have a shared common namespace for others if that may happen

Reply:
fixed, `akka.serialization.jackson`